### ISSUE TITLE
Price invert does not work

### ISSUE THREAD

**Issue by jon4hz** [miraclesu/uniswap-sdk-go#7]
Hey, the `Invert()` method for the Price type does not work. Shouldn't you invert the fraction and scalar, too? 

Something like:
```go
func (p *Price) Invert() *Price {
	return &Price{
		fraction:      p.fraction.invert(),
		baseCurrency:  p.quoteCurrency,
		quoteCurrency: p.baseCurrency,
		scalar:        p.scalar.invert(),
	}
}
```

Returning the price would also allow to chain this method with other methods like `ToSignificant()` which can be pretty useful.

**Comment by miraclesu** [comment]
Thank you for your feedback, it is fixed in v0.2.7

**Comment by jon4hz** [comment]
Nice, thanks a lot
